src/js/plugin/sdk/actions/MarathonActions.js

```diff
@@ -0,0 +1,30 @@
+import MarathonService from "../services/MarathonService";
+
+var service = new MarathonService();
+
+export default class MarathonActions {
+
+  getDeployments() {
+    return service.request({
+      resource: "/v2/deployments"
+    });
+  }
+
+  getGroup(group) {
+    /* ${group} already has the leading '/' */
```

Comment:
I'd add a check to make sure `group` actually starts with a slash `/`

Reply:
:+1: